### ISSUE TITLE
Reduce and label navigation landmarks for Header and Footer

### ISSUE THREAD

**Issue by developpeurweb** [SAP/spartacus#13511]
**Describe the bug**

- Automated testing detects way too many unnamed navigation landmarks
- When 2 or more of the same landmark exist, they should be labelled for differentiation.
- Screen Reader users are able to navigate by landmarks, therefore they should be reduced to a manageable number and have meaningful names.

**Tell us the version of Spartacus**
- Library version: 4.0.0.

**To Reproduce**
1. Open any Screen Reader (VO, NVDA, JAWS).
2. Bring up the Rotor or Element List Menu.
3. Choose Landmarks view.
4. See error.

**Expected behavior**

- Pass the automated test
- Use only predefined header and footer landmarks
https://wiki.wdf.sap.corp/wiki/display/spar/Landmarks

**Screenshots**
AMP Test
![Screen Shot 2021-08-05 at 12 25 56 PM](https://user-images.githubusercontent.com/15113219/130154011-54837369-0b1c-4854-adf6-f140ad62f6fb.png)
![Screen Shot 2021-08-05 at 7 07 58 PM](https://user-images.githubusercontent.com/15113219/130154031-ef4223e8-b636-40b9-b28d-570962ded818.png)
![Screen Shot 2021-08-05 at 12 08 18 PM](https://user-images.githubusercontent.com/15113219/130154047-9972b0f8-0bb2-4326-9fba-e756cc621a06.png)
![Screen Shot 2021-08-05 at 12 22 27 PM](https://user-images.githubusercontent.com/15113219/130154063-240c48e9-433a-43a2-ae4d-8d4266229717.png)

VoiceOver
![2021-08-05_12-24-25](https://user-images.githubusercontent.com/15113219/130153994-953a9375-bc58-4a3a-8691-b3fd6e9804d3.jpg)

NVDA
<img width="960" alt="2021-08-19_18-41-30" src="https://user-images.githubusercontent.com/15113219/130153930-54c4dfd2-8c92-4bb3-8006-b46758b6eb33.png">

**Comment by KateChuen** [comment]
Tested on develop branch of spartacus using yarn
More extensive testing (on other screenreaders and devices) will be done with RC.

<img width="877" alt="Screen Shot 2022-04-13 at 1 49 39 PM" src="https://user-images.githubusercontent.com/45409515/163240598-b515b00c-bb84-410e-81f7-64f1fb9b15a6.png">

<img width="877" alt="Screen Shot 2022-04-13 at 1 49 39 PM" src="https://user-images.githubusercontent.com/45409515/163240668-d0bd9280-d7ce-4d12-95af-8a52826e4e2e.png">
